freespace2/freespace.cpp

```diff
@@ -1052,6 +1053,7 @@ void game_level_init()
 	stars_pre_level_init();
 	neb2_level_init();
 	nebl_level_init();
+	Multi_Turret_Manager.reset(); // Cyborg, this can safely be done after everything else.  At some point, I'll probably consolidate these.
```

Comment:
This appears to be a cleanup function so it should really go in `game_level_close()` instead.